File: commonLib/src/main/java/com/travel/commonlib/activity/city/Hits.java

package com.travel.commonlib.activity.city;

import com.google.gson.annotations.Expose;
import com.google.gson.annotations.SerializedName;

import java.util.List;

public class Hits {

    @SerializedName("total")
    @Expose
    private long total;
    @SerializedName("max_score")
    @Expose
    private Object maxScore;
    @SerializedName("hits")
    @Expose
    private List<Hit> hits = null;

    public long getTotal() {
        return total;
    }

    public void setTotal(long total) {
        this.total = total;
    }

    public Object getMaxScore() {
        return maxScore;
    }

    public void setMaxScore(Object maxScore) {
        this.maxScore = maxScore;
    }

    public List<Hit> getHits() {
        return hits;
    }

    public void setHits(List<Hit> hits) {
        this.hits = hits;
    }

}

File: commonLib/src/main/java/com/travel/commonlib/hotel/elastic/Hits.kt
package com.travel.commonlib.hotel.elastic

import com.google.gson.annotations.Expose
import com.google.gson.annotations.SerializedName
import java.io.Serializable

class Hits : Serializable{
    @SerializedName("total")
    @Expose
    var total = 0

    @SerializedName("max_score")
    @Expose
    var maxScore = 0.0

    @SerializedName("hits")
    @Expose
    var hits: List<Hit>? = null
}